src/lib/routes/admin-api/project/variants.ts

```diff
@@ -112,6 +129,9 @@ export default class VariantsController extends Controller {
             handler: this.getVariantsOnEnv,
             middleware: [
                 openApiService.validPath({
+                    summary:
+                        'Get variants for a feature in a specific environment',
+                    description: `Returns the variants for a feature in a specific environment.`,
```

Comment:
What happens if you have no variants?

```suggestion
                    description: `Returns the variants for a feature in a specific environment. If the feature has no variants in this environment ...`,
```

Reply:
```suggestion
                    description: `Returns the variants for a feature in a specific environment. If the feature has no variants it will return an empty array of variants`,
```